use-cases.md

```diff
@@ -119,9 +134,28 @@ After a customer has checked out from a reservation, reputation management syste
 
+## Customer relationship management systems
+
+### Inital Data Pull
+
+Performed once when setting up the connection, because the CRM needs to obtain existing customers and previous reservations. The CRM should obtain the customers and reservations in time-limited batches using [Get all reservations](operations/reservations.md#get-all-reservations) and [Get all customers](operations/customers.md#get-all-customers) with the reservation time filter set to `Created` \(that will give you all customers and reservations which were created in the selected interval\). Size of the batches depends on size of the hotel and its occupancy, but in general **weekly batches** are recommended and should work well even for big hotels \(1000+ units\). In order to get all customers and reservations e.g. in the past year, the CRM should call [Get all customers](operations/reservations.md#get-all-customers) and [Get all reservations](operations/reservations.md#get-all-reservations) sequentially 52 times (one call for each week in the past year). That would give the CRM all customers and all reservations that have been created within the past year. To obtain products associated with reservations e.g. a breakfast, `Items` should be set to `true` in the [Extent](operations/reservations.md#reservation-extent) parameter.
+
+One can take advantage of the fact that reservations are usually booked a few weeks or months in advance. The further in the future, the lower the occupancy, so the reservation batch length may increase with the distance into future from current date. E.g. weekly batches can be used only for the first three months of the future year when there is higher occupancy. And for the remaining 9 months, monthly batches would be sufficient. This would reduce the operation count from 52 to 21 \(12 weekly batches + 9 monthly batches\).
+
+### Retrieving information
+
+A CRM should always be up to date with the latest data. Using the [Get all reservations](operations/reservations.md#get-all-reservations) and [Get all customers](operations/customers.md#get-all-customers) operations hourly with the `Updated` `TimeFilter` with an `EndUtc` of the current time and `StartUtc` with one hour before the current time would ensure you always have newly created and updated customers and reservations.
+
+### Marketing opt-out
+
+A customer has the ability to opt-out of marketing communication. As customer relationship management systems are also commonly used to launch marketing campaigns, the integration should acknowledge that in the [Customer Response](operations/customers.md#response), if a `Customer` has `SendMarketingEmails` within their `Options`, then the customer is subscribed to any marketing.
```

Comment:
Event management system is not written in the file anywhere

Reply:
@BrittonA Yup. Welcome to my first ever pull request attempt. Fixing it now